[placeholder] Sidecar Containers alpha feature (#41802)
kubernetes/website: content/en/docs/reference/command-line-tools-reference/feature-gates.md

| `SecurityContextDeny` | `false` | Alpha | 1.27 | |
| `ServiceNodePortStaticSubrange` | `false` | Alpha | 1.27 | 1.27 |
| `ServiceNodePortStaticSubrange` | `true` | Beta | 1.28 | |

> | `SidecarContainers` | `false` | Alpha | 1.28 | |

Reviewer: Remember to describe the feature as well.

  risk of collision. See
  [Avoiding collisions](/docs/concepts/services-networking/service/#avoiding-collisions)
  for more details.
- `SidecarContainers`: Allow `restartPolicy` to be set to `Always` when there are

>   sidecar containers in a pod. See

Reviewer 2: ```suggestion
  sidecar containers (restartable init containers) in a pod. See
```

Don't feel obligated to make this change if you don't think it helps.

> - `SidecarContainers`: Allow `restartPolicy` to be set to `Always` when there are
>   sidecar containers (restartable init containers) in a pod. See

Reviewer 3: ```suggestion
- `SidecarContainers`: Allow setting the `restartPolicy` of an init container to
  `Always` so that the container becomes a sidecar container (restartable init containers).
  See
```
